src/subscription_manager/managercli.py

```diff
@@ -1270,6 +1272,11 @@ def _validate_options(self):
             system_exit(os.EX_USAGE, _("Error: Activation keys do not allow environments to be specified."))
         elif self.autoattach and self.options.activation_keys:
             system_exit(os.EX_USAGE, _("Error: Activation keys cannot be used with --auto-attach."))
+        elif self.autoattach and self.options.disable_autoattach:
+            system_exit(
+                os.EX_USAGE,
+                _("Error: The --disable-auto-attach option cannot be used with the --auto-attach option'.")
+            )
```

Comment:
Don't we also need a check that we are using an activation key when we do the disable autoattach?

Reply:
Yeah, good point, the messaging could be better here. Because it is not possible to use `--auto-attach` with activation key. I will change it.